src/adder.rs

```diff
@@ -1,39 +1,55 @@
+#[path = "./table_header_adder.rs"]
```

Comment:
```suggestion
```

Reply:
ohhhhhh then you should move the file to `src/adder/table_header_adder.rs`

you almost never want to use `#[path = ...]` on a mod declaration